src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -80,9 +89,16 @@
 
     public Configuration(){}
 
+    @Deprecated
+    public Configuration(CharSource staticSource, CharSink staticSink){
```

Comment:
And you can do that with the Charsource / CharSink. In fact it will be a much nicer API. You just do `Config.load` and it will "magically" load the config from the server, without the config class having to know anything about packets or anything, it all happens in the CharSource.